<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingWidget</class>
 <widget class="QWidget" name="SettingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame_2">
     <property name="styleSheet">
      <string notr="true">QFrame#frame_2 {
	background-color: rgb(245, 245, 245);
	border-radius: 51;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>150</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>150</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QPushButton" name="banButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>17</pointsize>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgb(245, 245, 245);
	color: rgb(60, 45, 50);
	border: 5px solid rgb(60, 45, 50);
	border-radius: 20;
}

QPushButton:hover {
	background-color: rgb(60, 45, 50);
	color: rgb(245, 245, 245);
}</string>
        </property>
        <property name="text">
         <string>Блокировка</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="notifyButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>17</pointsize>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgb(245, 245, 245);
	color: rgb(60, 45, 50);
	border: 5px solid rgb(60, 45, 50);
	border-radius: 20;
}

QPushButton:hover {
	background-color: rgb(60, 45, 50);
	color: rgb(245, 245, 245);
}</string>
        </property>
        <property name="text">
         <string>Уведомления</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="complaintButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>17</pointsize>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgb(245, 245, 245);
	color: rgb(60, 45, 50);
	border: 5px solid rgb(60, 45, 50);
	border-radius: 20;
}

QPushButton:hover {
	background-color: rgb(60, 45, 50);
	color: rgb(245, 245, 245);
}</string>
        </property>
        <property name="text">
         <string>Жалоба</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="aboutButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>17</pointsize>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgb(245, 245, 245);
	color: rgb(60, 45, 50);
	border: 5px solid rgb(60, 45, 50);
	border-radius: 20;
}

QPushButton:hover {
	background-color: rgb(60, 45, 50);
	color: rgb(245, 245, 245);
}</string>
        </property>
        <property name="text">
         <string>О LinkDove</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="quitButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>75</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>17</pointsize>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: rgb(245, 245, 245);
	color: rgb(60, 45, 50);
	border: 5px solid rgb(60, 45, 50);
	border-radius: 20;
}

QPushButton:hover {
	background-color: rgb(60, 45, 50);
	color: rgb(245, 245, 245);
}</string>
        </property>
        <property name="text">
         <string>Выйти из аккаунта</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
